tests/unit_test_config_generator.py

```diff
@@ -189,7 +189,6 @@ def test_invalid_draft_style(self):
 
         expected = copy.deepcopy(isi_schema)
         del expected['properties']['disconnected_nodes']['items']['required']
-        expected['required'] = ['disconnected_nodes']
```

Comment:
If `isi_schema` is converted to draft 4 json schema won't 

`expected['required'] = ['disconnected_nodes']` 

be in the result?